### ISSUE TITLE
Could not resolve pagination key in template data

### ISSUE THREAD

**Issue by paulrobertlloyd** [11ty/eleventy#171]
Having come back to my implementation using 11ty, and upgrading to v0.5, I’m now getting this error:

```
Problem writing eleventy templates (more info in DEBUG output): 
Error: Could not resolve pagination key in template data: collections.checkins
    at Pagination._resolveItems (/Users/paulrobertlloyd/Sites/v4.paulrobertlloyd.com/node_modules/@11ty/eleventy/src/Plugins/Pagination.js:80:11)
    at Pagination.setData (/Users/paulrobertlloyd/Sites/v4.paulrobertlloyd.com/node_modules/@11ty/eleventy/src/Plugins/Pagination.js:43:22)
    at new Pagination (/Users/paulrobertlloyd/Sites/v4.paulrobertlloyd.com/node_modules/@11ty/eleventy/src/Plugins/Pagination.js:9:8)
    at Template.getTemplates (/Users/paulrobertlloyd/Sites/v4.paulrobertlloyd.com/node_modules/@11ty/eleventy/src/Template.js:526:21)
    at TemplateMap.populateUrlDataInMap (/Users/paulrobertlloyd/Sites/v4.paulrobertlloyd.com/node_modules/@11ty/eleventy/src/TemplateMap.js:85:38)
```

This is my front matter:

```
layout: layouts/index
title: Checkins
summary: Places I've been.
type: checkin
pagination:
  data: collections.checkins
  size: 12
```

This is the essential part of my index template:

```
{% for item in pagination.items -%}
  …
{%- endfor %}
```

What’s changed for this to start failing? Do I need to update something?

**Comment by zachleat** [comment]
Well, crap. Sorry about that. I'm surprised though, I have tests around this. I'll take another look.

**Comment by zachleat** [comment]
@paulrobertlloyd Where is `collections.checkins` coming from? Your config file? A collection generated from `tags`?

**Comment by paulrobertlloyd** [comment]
From my config file:

```
eleventy.addCollection('checkins', collection => {
    return collection.getFilteredByGlob('**/checkins/!(index)*.md').reverse();
  });
```

This is happening for all of my collections for which I’m then attempting to paginate.

**Comment by zachleat** [comment]
Alright yeah, I can confirm this is a regression due to #160 and should affect only collections specified in the configuration api (not those generated by eleventy using tags).

**Comment by zachleat** [comment]
Whew, okay—I added a bunch more tests around this. This will be included with 0.5.1.
